modules/ROOT/pages/_partials/commons/common-indexing.adoc

```diff
@@ -201,14 +201,16 @@ The syntax for array index configuration is shown below:
 |`path`
 |image:ROOT:no.png[]
 |Path to the array to be indexed.
+Use `"[]"` to represent a property that is an array of each nested array level.
```

Comment:
are the "" quotes required? (I realise this is from the previous version, so not an urgent comment either way)

Reply:
(Perhaps worth adding an example to clarify what this looks like?
Again, as future fix)